org_fedora_oscap/common.py

```diff
@@ -73,12 +74,12 @@ def N_(string): return string
 
 SSG_DIR = "/usr/share/xml/scap/ssg/content/"
 SSG_CONTENT = "ssg-rhel7-ds.xml"
-if constants.shortProductName != 'anaconda':
-    if constants.shortProductName == 'fedora':
+if shortProductName != 'anaconda':
```

Comment:
It would be better to get rid of this constant completely. The OSCAP add-on should call `get_os_release_value("ID")` to get a lower-case identifier of the operating system. It will return better results for other products, like centos.